types/src/proof/proptest_proof.rs

```diff
@@ -15,20 +15,26 @@ use crypto::{
 use proptest::{collection::vec, prelude::*};
 use rand::{seq::SliceRandom, thread_rng};
 
+fn arb_non_placeholder_accumulator_sibling() -> impl Strategy<Value = HashValue> {
+    any::<HashValue>().prop_filter("Filter out placeholder sibling.", |x| {
+        *x != *ACCUMULATOR_PLACEHOLDER_HASH
+    })
+}
+
+fn arb_accumulator_sibling() -> impl Strategy<Value = HashValue> {
+    prop_oneof![
```

Comment:
Do you want to use the [weighed variant](https://docs.rs/proptest/0.9.4/proptest/macro.prop_oneof.html) of `oneof` here?

Reply:
I think uniform weight is okay here...